diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
index fbf3992d6b6016ccd142e9a1a39f0533f564a7fc..281217562630eec0e258ea6eebe887b872447925 100644
--- a/components/permissions/permission_request_manager.cc
+++ b/components/permissions/permission_request_manager.cc
@@ -82,6 +82,7 @@ bool IsArOrCameraRequest(RequestType type) {
 }
 
 bool ShouldGroupRequests(PermissionRequest* a, PermissionRequest* b) {
+  BRAVE_SHOULD_GROUP_REQUESTS
   if (a->GetOrigin() != b->GetOrigin())
     return false;
 
