diff --git a/components/permissions/request_type.h b/components/permissions/request_type.h
index d219ccbc3cbda2843ff24ade4d3c4cd04b29227b..4daf0d72e116af2a503829d85d0c17bbf06d24cc 100644
--- a/components/permissions/request_type.h
+++ b/components/permissions/request_type.h
@@ -49,6 +49,7 @@ enum class RequestType {
   kSecurityAttestation,
 #endif
   kStorageAccess,
+  BRAVE_REQUEST_TYPES
   kVrSession,
 #if !defined(OS_ANDROID)
   kWindowPlacement,
