diff --git a/components/permissions/permission_uma_util.cc b/components/permissions/permission_uma_util.cc
index 800f537d9e4480a15d73e854709f0d503c3a3c6a..3d0cbc67250b123abebb239ae86c23683bbbb05a 100644
--- a/components/permissions/permission_uma_util.cc
+++ b/components/permissions/permission_uma_util.cc
@@ -107,6 +107,7 @@ RequestTypeForUma GetUmaValueForRequestType(RequestType request_type) {
     case RequestType::kWindowPlacement:
       return RequestTypeForUma::PERMISSION_WINDOW_PLACEMENT;
 #endif
+    BRAVE_GET_UMA_VALUE_FOR_REQUEST_TYPE
   }
 }
 
@@ -762,6 +763,7 @@ void PermissionUmaUtil::RecordPermissionAction(
       base::UmaHistogramEnumeration("Permissions.Action.FileHandling", action,
                                     PermissionAction::NUM);
       break;
+    BRAVE_RECORD_PERMISSION_ACTION
     // The user is not prompted for these permissions, thus there is no
     // permission action recorded for them.
     default:
