diff --git a/components/permissions/permission_request_manager.h b/components/permissions/permission_request_manager.h
index 4442cb80216623362475597a5e0ee6a868b87f7e..1c4bbefff131702fd722f3b88678a20acf4422d9 100644
--- a/components/permissions/permission_request_manager.h
+++ b/components/permissions/permission_request_manager.h
@@ -193,6 +193,7 @@ class PermissionRequestManager
   current_request_prompt_disposition_for_testing() {
     return current_request_prompt_disposition_;
   }
+  BRAVE_PERMISSION_REQUEST_MANAGER_H_
 
  private:
   friend class test::PermissionRequestManagerTestApi;
